db141Wiki.php

```diff
@@ -6,7 +6,7 @@
 	$getLanguageCode = 'getLanguageCode';
 	$getTranslation = 'getTranslation';
 
-	http_response_code( 404 );
+	http_response_code( 503 );
```

Comment:
FYI, I //believe// I changed this to `500` already on-servers but never updated the other PR.

Reply:
It would appear so, I see status 500 being reported by the webserver